### DISCUSSION TITLE
Provider Maintenance

### DISCUSSION THREAD

**Discussion by sergeiwaigant** [Telmate/terraform-provider-proxmox#991] [Q&A] [answered]
Hey guys.

In one project I am evaluating if we need to migrate to another Terraform Provider for Proxmox.
Background is the concern, that this provider is not maintained anymore - is that true? I see frequent Commits and Pull Requests, but the RC1 is already 3 month old.
Does the provider support Proxmox 8?

Furthermore I am not able to find anything about the Telmate LLC which gives some more trust.

Any comments on this from the contributors?

Regards
Sergei

**Comment by Tinyblargon** [comment, marked as answer]
Hi @sergeiwaigant currently @mleone87 @hestiahacker and I maintain the provider with the very limited free time we have. Telmate LLC does not exist anymore but we still have the power to add new maintainers, (we can't use the projects tab as far as I'm aware).

Proxmox 8 is supported (all testing has been done with pve7 and pve8).

#993 Is tracking all blockers for RC2

**Reply by sergeiwaigant** [reply]
Thank you for the fast response!

I guess that the testing with Proxmox 8 has been done with the v3 RC1, not with the latest v2?
